### ISSUE TITLE
Unable to add background image in the Header

### ISSUE THREAD

**Issue by oswaldderiemaecker** [clawfire/lean-template#3]

**Comment by clawfire** [comment]
There's no support for this actually. What to you want to achieve ?

**Comment by clawfire** [comment]
That's not something supported out of the box but I don't find a way de unregister thoses settings from the customiser. There's no support for this actually. What to you want to achieve ?

**Comment by oswaldderiemaecker** [comment]
Every themes support background images no ?

**Comment by clawfire** [comment]
No. I mean it's need to be implemented. It's not working like magic :D as I said, I don't find the way to de-activate the background option in the customizer (the panel where you can set it) but it's not implemented yet. If you tell me how you want to use it (where you want to be able to add a background picture) I can implement it :)
